fets/model/project/Dockerfile

```diff
@@ -12,11 +17,13 @@ LABEL status="beta"
 RUN apt-get -y update && apt -y full-upgrade && apt-get -y install apt-utils wget git tar build-essential curl nano
 
 # install all python requirements
-WORKDIR /workspace
+# TODO I chose a different name than "/workspace" to avoid confusion with the MLCube workspace folder mounted at runtime. OK?
```

Comment:
No problem, one requirement is that there should be a workdir in the root folder for using the conversion from docker to singularity, so any name could work.